using_images/sti_images/python.adoc

```diff
@@ -264,8 +269,14 @@ openshift/python-33-rhel7
 To set these environment variables, you can place them into `.sti/environment`
 file inside your source code repository.
 
-* [envvar]#APP_FILE# - This variable specifies file name (defaults to `app.py`) passed to the python interpreter which is responsible for launching application.
+* `*APP_FILE*` - This variable specifies file name (defaults to *_app.py_*) passed to the python interpreter which is responsible for launching the application.
```

Comment:
Should this be "This variable specifies the file name (blah) which is passed to..."?